README.rst

```diff
@@ -16,8 +16,8 @@
    :target: https://www.codacy.com/app/leponteineptique/MyCapytain
 
 
-MyCapytain is a python library which provides a large set of methods to interact with Text Services API  such as the \
-Canonical Text Services, the Distributed Text Services. It also provides a programming interface to exploit local \
+MyCapytain is a python library which provides a large set of methods to interact with CapitainsCtsText Services API  such as the \
+Canonical CapitainsCtsText Services, the Distributed CapitainsCtsText Services. It also provides a programming interface to exploit local \
```

Comment:
Did you really want to change Text here to CapitainCtsText? I would think that you actually want to leave these as Canonical Text Services"